specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp

```diff
@@ -9,6 +9,7 @@ using TypeSpec.Rest;
 using TypeSpec.Versioning;
 using Azure.Core;
 
+#suppress "@azure-tools/typespec-azure-core/auth-required" "MUST fix in next update"
```

Comment:
Contoso.WidgetManager is literally just an example, so this one I think we should just fix. 
```suggestion
@useAuth(AadOauth2Auth<["https://contoso.azure.com/.default"]>)
```

Reply:
+1 on that. If this is actually just an example we should try to not show suppression

Reply:
The only remaining suppression in this example is temporary and required until https://github.com/Azure/typespec-azure/issues/3107 is fixed.